packages/web3-shared/src/assets/chains.json

```diff
@@ -37,6 +38,7 @@
             "decimals": 18
         },
         "faucets": ["https://faucet.ropsten.be?${ADDRESS}"],
+        "features": [],
```

Comment:
ropsten supports eip-1159, it's suitable to test 1159 by it.

Reply:
DONE